### PR TITLE
Pretty

### PR DESCRIPTION
**Pull request by petean09**
Changed some styling and added pie chart

### CODE REVIEW

File: server/services/transcation-service.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
+const Transaction = require('../models/transaction-model');
+const ReadPreference = require('mongodb').ReadPreference;
+
```
Comment (rublan01) [inline server/services/transcation-service.js]: we need to probably delete this file? (the typo in the name) I think the imports are fixed in other places so we should be fine to do that :)

Comment (rublan01) [reply server/services/transcation-service.js]: (we have transcation service and transaction service lol)

File: server/services/transcation-service.js [hunk 2]
```diff
@@ -0,0 +1,114 @@
+const Transaction = require('../models/transaction-model');
```
Comment (kuehma02) [inline server/services/transcation-service.js]: This is transcation-service, (spelled wrong, i think we can delete this)

File: server/services/transcation-service.js (hunk 1 above)
Comment (kuehma02) [reply server/services/transcation-service.js]: Lol I made the same review lmao

Comment (petean09) [reply server/services/transcation-service.js]: Deleted it

File: server/services/transcation-service.js (hunk 2 above)
Comment (petean09) [reply server/services/transcation-service.js]: Deleted it

**Pull request merged**